### PR TITLE
Completed Stride() documentation

### PR DESCRIPTION
**Pull request by ujjwal-9**
Documented full functionality of stride() as defined [here](https://github.com/pytorch/pytorch/blob/312e0ce3ba1fa74854a1fc82aa1b7c24baeb4006/torch/lib/TH/generic/THTensor.c#L28)
issue #2946 resolved

### CODE REVIEW

**Review by soumith — APPROVED** [review]

**Comment by Ujjwal-9** [comment]
@soumith I have corrected the PEP8 Error but still it shows same error. do you mind having a look at the commit. if its consistent or not.

**Comment by soumith** [comment]
@Ujjwal-9 it's saying that where you are indenting the lines, you are using `[tab]` characters instead of `[space]`. Can you change all the tabs to spaces in your changes?

File: torch/_tensor_docs.py [hunk 1]
```diff
@@ -1453,9 +1453,24 @@ def callable(a, b) -> number
...
 Returns the stride of the tensor.
+Stride is the jump necessary to go from one element to the next one in the specified dimension dim.
+Tuple is returned when no Argument is passed. So we get stride in all dimensions.
+Integer value is returned when we desire stride in particular dimension.
+
+Args:
+    dim (int): The desired dimension in which stride is required.
+
+Example:
+>>> x = torch.Tensor([[1, 2, 3, 4, 5], [6, 7, 8, 9, 10]])
```
Comment (soumith) [inline torch/_tensor_docs.py]: you need to replace the tabs with 4 spaces. right now you removed the spaces entirely.

File: torch/_tensor_docs.py [hunk 2]
```diff
@@ -1634,9 +1649,9 @@ def callable(a, b) -> number
 An additional dimension of size size is appended in the returned tensor.
 
 Args:
-    dim (int): dimension in which unfolding happens
-    size (int): size of each slice that is unfolded
-    step (int): the step between each slice
+ dim (int): dimension in which unfolding happens
```
Comment (soumith) [inline torch/_tensor_docs.py]: same as above. you need to replace the tabs with 4 spaces. right now you removed the spaces entirely.

File: torch/_tensor_docs.py (hunk 1 above)
Comment (Ujjwal-9) [reply torch/_tensor_docs.py]: :smile: this time i cleared all the tests. but i have pushed the above said changes.

Comment (Ujjwal-9) [reply torch/_tensor_docs.py]: This seems to be a bug. Should I open an issue regarding this.

**Comment by Ujjwal-9** [comment]
@soumith issue fixed now please merge this PR

**Comment by soumith** [comment]
looks great. thank you!

**Pull request merged**